qualtran/bloqs/arithmetic.py

```diff
@@ -212,3 +219,74 @@ def t_complexity(self):
         # See: https://github.com/quantumlib/cirq-qubitization/issues/219
         # See: https://github.com/quantumlib/cirq-qubitization/issues/217
         return TComplexity(t=8 * self.bitsize)
+
+
+@frozen
+class LessThanEqual(Bloq):
+    r"""Implements $U|x,y,z\rangle = |x, y, x \le y\rangle$.
```

Comment:
z has dissapeared in your telling. Either change to |x, y, 0> -> x, y, x<=y or a z \oplus